test/integration/replicaset/replicaset_test.go

```diff
@@ -112,32 +112,6 @@ func newMatchingPod(podName, namespace string) *v1.Pod {
 	}
 }
 
-// verifyRemainingObjects verifies if the number of the remaining replica
-// sets and pods are rsNum and podNum. It returns error if the
-// communication with the API server fails.
-func verifyRemainingObjects(t *testing.T, clientSet clientset.Interface, namespace string, rsNum, podNum int) (bool, error) {
```

Comment:
@tianshapjq thanks for cleaning this up!

can you remove for the replication controller as well?

https://github.com/kubernetes/kubernetes/blob/0fcc38425843d973538ce743a907e4c98095f8a0/test/integration/replicationcontroller/replicationcontroller_test.go#L111

Reply:
@nikhita sure thing~